src/main/kotlin/org/jitsi/jibri/Main.kt

```diff
@@ -113,13 +66,21 @@ fun main(args: Array<String>) {
             }
         }
     }
+    val jibriId = configSupplier<String> {
```

Comment:
Intentionally not using the `config` delegate?

Reply:
So I thought it was because you can't use delegates on non-members, but I just checked and `by lazy` does work, but the config ones complain.  I'll have to look into that at some point.